pyproject.toml

```diff
@@ -8,5 +8,8 @@ select = ["E", "F", "W", "PIE", "C4", "ISC", "RUF100", "A"]
 ignore = ["W292", "E741", "E402", "C408", "ISC003"]
 flake8-implicit-str-concat.allow-multiline=false
 
+[tool.ruff.lint.flake8-tidy-imports.banned-api]
+"pytest.main".msg = "pytest.main requires special handling that is easy to mess up!"
```

Comment:
good catch! let's add this to openpilot too

Reply:
we added it here when we had similar problems in `test_onroad`: https://github.com/commaai/openpilot/pull/31623